alerting/src/main/kotlin/org/opensearch/alerting/model/destination/Destination.kt

```diff
@@ -260,7 +260,7 @@ data class Destination(
             }
             DestinationType.CUSTOM_WEBHOOK -> {
                 destinationMessage = LegacyCustomWebhookMessage.Builder(name)
-                    .withUrl(getLegacyCustomWebhookMessageURL(customWebhook))
+                    .withUrl(getLegacyCustomWebhookMessageURL(customWebhook, compiledMessage))
                     .withHeaderParams(customWebhook?.headerParams)
                     .withMessage(compiledMessage).build()
```

Comment:
The message is passed in here. It's not required in `getLegacyCustomWebhookMessageURL` since that method is only combining the individual URL parameters to a String to pass to `withUrl`.

Reply:
Yes that is correct. It is true that it is not required for actual construction of URI, but unfortunately it is required field of `LegacyCustomWebhookMessage` and it always throws `IllegalArgumentException("Message content is missing")`. I went with this solution as it is simple and this legacy destination is only used in limited number of cases (and it will probably be removed in some of future releases).

It would probably be best to completely separate URI construction code of constructing `LegacyCustomWebhookMessage` object itself, but as message is available here I went with this easy fix.